imager/README.txt

```diff
@@ -1,33 +1,42 @@
 Raspberry Pi 4 EEPROM bootloader rescue image
 *********************************************
 
-The Raspberry Pi 4 has a small EEPROM used to store the bootloader.
+The Raspberry Pi 4 contains a small EEPROM used to store the bootloader.
 
 This rescue image reverts the bootloader EEPROM to factory default settings.
 
-This rescue image also updates the USB 3.0 (VL805) firmware to the latest
-version (138a1) with better full-speed Isochronous endpoint support.
+This rescue image also updates the USB 3 controller (VL805) firmware to the
+latest version, 138a1, which has better full-speed isochronous endpoint 
+support.
 
-This easiest method for creating EEPROM rescue images or formatting sdcards
-is to use the Raspberry Pi Imager from https://raspberrypi.com/software.
-The imager provides a GUI for downloading the latest version of this rescue
-image and flashing it to a spare SD CARD.
```

Comment:
Restore “spare” because this will deltas whatever was on the card. So we always use the word spare to imply that